app/pods/components/notification-dropdown/component.js

```diff
@@ -24,13 +25,22 @@ export default class NotificationDropdownComponent extends Component {
 
   constructor () {
     super (...arguments)
+    const _this = this;
 
     this.get('loadNotifications').perform ()
 
     setInterval (() =>
       this.get ('loadNotifications').perform (),
       this.get('refreshInterval')
     )
+    document.addEventListener("click", function() {
```

Comment:
Move this code inside [`didInsertElement`](https://guides.emberjs.com/release/components/the-component-lifecycle/#toc_integrating-with-third-party-libraries-with-didinsertelement) hook.